src/main/java/appeng/container/slot/AppEngCraftingSlot.java

```diff
@@ -19,12 +19,13 @@
 package appeng.container.slot;
 
 
+import appeng.util.inv.WrapperInvItemHandler;
 import net.minecraft.entity.player.EntityPlayer;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Wrong order for 'net.minecraft.entity.player.EntityPlayer' import. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck)